src/common/formUtils.ts

```diff
@@ -35,7 +35,11 @@ export const commonYupValidation = {
   nickname: Yup.string()
     .required("Stage name is required")
     .matches(/^[aA-zZ\s]+$/, "Please only use letters"),
-  newPassword: Yup.string().matches(
+  newPassword: Yup.string().test(
+    "is-trimmed",
+    "Password must not start or end with a space character",
+    (password) => password ? password === password?.trim() : false
+  ).matches(
```

Comment:
I sneaked this into this PR since it was fairly small, it tests for start/end space characters on new password field.